apps/remix-ide/src/app/components/plugin-manager-component.js

```diff
@@ -1,73 +1,11 @@
-import { IframePlugin, ViewPlugin, WebsocketPlugin } from '@remixproject/engine-web'
+import { ViewPlugin } from '@remixproject/engine-web'
 import { PluginManagerSettings } from './plugin-manager-settings'
```

Comment:
Is this file still being used? If not, remove the import statement.

Reply:
It is being used in the permissions component through PluginManagerComponent